src/action.rs

```diff
@@ -41,6 +55,13 @@ pub enum Action {
         /// The tap action.
         tap: &'static Action,
     },
+    /// A sequence of KeyEvents
+    Sequence {
+        /// How long to delay between events
+        delay: u16, // NOTE: Currently unused
+        /// The sequence of KeyEvents that will be triggered
+        actions: &'static [SequenceEvent],
```

Comment:
That's not actions, thats keys. `actions` would be if you had `&[Action]`.

Reply:
Good catch.  I've renamed it to just `events` in my working code.